DNN Platform/Library/Data/DataProvider.cs

```diff
@@ -451,12 +451,6 @@ public virtual string GetUnappliedIterations(string version)
 
 		#region Host Settings Methods
 
-		public virtual void AddHostSetting(string SettingName, string SettingValue, bool SettingIsSecure,
```

Comment:
Is there not a backwards compatibility concern with removing these methods (and their corresponding stored procedures)?